src/events/names.rs

```diff
@@ -0,0 +1,140 @@
+/// BEWARE: This file has been automatically generated by the `atspi-codegen` crate.
+/// Any edits to this file will be overwritten by the next re-generation of this file.
+
+use serde::{Serialize, Deserialize};
+use enumflags2::bitflags;
+
+pub trait Signal {
+  const INTERFACE: &'static str;
+}
+#[bitflags]
```

Comment:
We likely want to annotate this enum with  #[non_exhaustive]. If it is even remotely possible the enum to change during its life, it should be. Adding #[non_exhaustive] allows us to add/remove variants while staying backwards compatible and  without breaking other peoples code.

See for reference: [Prevent breaking changes with non_exhaustive](https://blog.knoldus.com/prevent-breaking-code-changes-in-future-releases-using-non-exhaustive-enums-in-rust/)

Reply:
(Of course the same goes for all other public enums here.)